tests/unit/resources/test_omnistack_clusters.py

```diff
@@ -112,6 +112,20 @@ def test_get_connected_clusters(self, mock_get):
         mock_get.assert_has_calls([call('/omnistack_clusters/12345/connected_clusters'),
                                   call('/omnistack_clusters?case=sensitive&id=12345&limit=500&offset=0&order=descending&sort=name')])
 
+    @mock.patch.object(Connection, "get")
+    def test_get_throughput_without_filters(self, mock_get):
+        cluster_data = {'name': 'name1', 'id': '12345'}
+        cluster = self.clusters.get_by_data(cluster_data)
+        cluster.get_throughput()
```

Comment:
Need to verify return object for the get throughput method

Reply:
Updated as suggested